contracts/Admin/VBNBAdmin.sol

```diff
@@ -91,6 +91,17 @@ contract VBNBAdmin is ReentrancyGuardUpgradeable, AccessControlledV8, VBNBAdminS
         emit ReservesReduced(reduceAmount);
     }
 
+    /**
+     * @notice Sets the interest rate model of the vBNB contract
+     * @param newInterestRateModel Address of the new interest rate model
+     * @custom:access Controlled by ACM
+     */
+    function setInterestRateModel(address newInterestRateModel) external {
+        _checkAccessAllowed("_setInterestRateModel(address)");
+        uint256 err = vBNB._setInterestRateModel(newInterestRateModel);
+        require(err == 0, "setting interest rate model failed");
```

Comment:
I would not require anything here, I would just return the value returned by `vBNB._setInterestRateModel`, as we are doing in the fallback function